src/Entity/OgMembership.php

```diff
@@ -219,10 +219,14 @@ public function revokeRoleById($role_id) {
    * {@inheritdoc}
    */
   public function getRoles() {
+    $roles = [];
+
     // Add the member role.
-    $roles = [
-      OgRole::getRole($this->getGroupEntityType(), $this->getGroup()->bundle(), OgRoleInterface::AUTHENTICATED),
-    ];
+    if ($group = $this->getGroup()) {
```

Comment:
👍 Worth a comment to explain why we do this check

Reply:
Sure!